README.md

```diff
@@ -97,9 +103,16 @@ Otherwise, the following steps will work for any language:
 
 ## Detailed usage
 
-### Enhancing streaming video with asbplayer features
+To use asbplayer, you can either choose to install the Chrome [extension](https://chromewebstore.google.com/detail/asbplayer-language-learni/hkledmpjpaehamkiehglnbelcpdflcab) or use the [webtool](https://killergerbah.github.io/asbplayer/). 

```

Comment:
Let's use the more common word "website" and not "webtool."
I think we can also explain when to use the extension or when to use the website e.g:
"To use asbplayer with streaming video, install the Chrome extension. Otherwise, use the website."